nautobot/extras/models/jobs.py

```diff
@@ -824,16 +824,23 @@ def log(
 
         message = sanitize(str(message))
 
+        obj_absolute_url = None
+        if obj is not None and hasattr(obj, "get_absolute_url"):
+            try:
+                absolute_url = obj.get_absolute_url()
+                if len(absolute_url) <= JOB_LOG_MAX_ABSOLUTE_URL_LENGTH:
+                    obj_absolute_url = absolute_url
+            except Exception:
+                pass
```

Comment:
```suggestion
            try:
                absolute_url = obj.get_absolute_url()
                # If absolute_url is longer than we can store, we discard it instead of truncating,
                # since a truncated URL is pretty useless.
                if len(absolute_url) <= JOB_LOG_MAX_ABSOLUTE_URL_LENGTH:
                    obj_absolute_url = absolute_url
            except Exception:
                pass
```